website/src/content/blog/2025-03-31-typespec-1-0-release/type_spec_1_0_release.md

```diff
@@ -0,0 +1,202 @@
+
+TypeSpec's core value proposition is the ability to define your API once and generate multiple output formats. The 1.0-RC release ships with a stable OpenAPI 3.0 emitter (`@typespec/openapi3`), enabling you to seamlessly integrate TypeSpec into existing OpenAPI workflows and toolchains.
+
+But our emitter story doesn't stop there. The TypeSpec ecosystem includes several additional emitters:
+
+- **OpenAPI 2.0 Emitter**: For teams that still require Swagger/OpenAPI 2.0 compatibility
+- **REST Emitter**: Produces JSON descriptions of REST APIs
+- **Protocol Buffer Emitter**: Generates Protocol Buffer (protobuf) definitions
+- **JsonSchema Emitter**: Creates JSON Schema documents from your TypeSpec types
+
+While some of these emitters remain in preview as we gather feedback, they demonstrate the extensibility of TypeSpec across different API description formats.
+
+For teams with existing OpenAPI definitions looking to adopt TypeSpec, we've also created the OpenAPI to TypeSpec converter tool. This tool helps you migrate your existing API definitions to TypeSpec, providing a smoother onboarding experience and preserving your investment in OpenAPI. You can find the converter in our [GitHub repository](https://github.com/microsoft/typespec/tree/main/packages/openapi3-to-typespec).
+
+For teams with specialized needs, TypeSpec 1.0-RC includes a new [emitter framework](https://github.com/alloy-framework/alloy) that allows you to build custom emitters. This framework provides access to TypeSpec's compiler APIs and type system, enabling you to transform your TypeSpec definitions into any format your organization requires.
+
+We'll be stabilizing both additional emitters and the emitter framework based on your feedback in upcoming releases. If you're interested in creating custom emitters, check out our [documentation](https://typespec.io/docs/emitters/overview) for guidance and examples.
+
+## Stability of core components
+
+The core components of TypeSpec, including the compiler, HTTP library, OpenAPI emitters, and VS/VSCode extensions, are in a stable state and ready for production use. These components have been thoroughly tested and are designed to provide a reliable foundation for your API development workflows.
+
+## Preview features and known gaps
+
+While the core components are stable, some features, such as the codegen emitters and certain protocol emitters, are still in preview. Here are the known gaps:
+
+- **Service code generation**:
+
+  - Missing support for advanced features like authentication and complex data transformations.
+  - Limited documentation and examples.
+
+- **Emitters in preview**:
+  - Protocol Buffer Emitter.
+  - JsonSchema Emitter.
+  - REST Emitter.
+
+We are actively seeking feedback on these preview features to prioritize fixes and enhancements. Your input is invaluable in shaping the future of TypeSpec.
+
+## 1.0 RC packages
+
+The following packages are considered stable and are part of the 1.0-RC release:
```

Comment:
Same note as above, stable implies 1.0, RC is very close to stable but not yet stable. If we find no issues it will become the stable release.